apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Snippets/AspNetSnippets.cs

```diff
@@ -29,5 +32,27 @@ public static void Register(HttpConfiguration config)
                 CloudErrorReportingExceptionLogger.Create(projectId, serviceName, version));
         }
         // End sample
+
+        // Sample: InitializeTrace
+        public class Global : HttpApplication
+        {
+            void Application_Start(object sender, EventArgs e)
+            {
+                string projectId = "[Google Cloud Platform project ID]";
+                // Trace a sampling of incoming Http requests.
+                CloudTrace.Initialize(projectId, this);
+            }
+        }
+        // End sample
+
+        // Sample: UseTracer
```

Comment:
Is it important that this sample includes the method declaration itself? Most of our samples don't - it makes sense for InitializeTrace for context, but I'm not sure it does in this case.